packages/react/src/LogoSuite/LogoSuite.test.tsx

```diff
@@ -0,0 +1,246 @@
+import React, {render, cleanup} from '@testing-library/react'
+import '@testing-library/jest-dom'
+
+import {LogoSuite} from './LogoSuite'
+import '../test-utils/mocks/match-media-mock'
+import {axe, toHaveNoViolations} from 'jest-axe'
+
+expect.extend(toHaveNoViolations)
+
+describe('LogoSuite', () => {
+  const mockHeading = 'Mock heading'
+  const mockDescription = 'Mock description'
+
+  afterEach(() => {
+    cleanup()
+    jest.clearAllMocks()
+  })
+
+  it('renders correctly into the document with default options', () => {
+    const testId = 'logobar'
+    const {getByText, getByTestId} = render(
+      <LogoSuite>
+        <LogoSuite.Heading>{mockHeading}</LogoSuite.Heading>
+        <LogoSuite.Description>{mockDescription}</LogoSuite.Description>
+        <LogoSuite.Logobar data-testid={testId}>
+          <svg data-testid="svg1" />
+          <svg data-testid="svg2" />
+        </LogoSuite.Logobar>
+      </LogoSuite>,
+    )
+
+    expect(getByText(mockHeading)).toBeInTheDocument()
+    expect(getByText(mockDescription)).toBeInTheDocument()
+    expect(getByTestId(testId).children.length).toBe(2)
```

Comment:
> are setting the minimum logo requirements to be 2?

No, this is just a unit test assertion checking that the two logos i'm trying to render do in fact appear correctly, no more, no less. 

There's currently no logic around how many logos you can have in LogoSuite. Do we need to set that on our end?

Reply:
Thank you for explaining this and sharing with me how the code breaks down, I really appreciate it! I agree, I don't think we need test cases since I can see some instances where designers or engineers in the future find more fit to use more rows (if we have more sponsors/logos). A recommendation for interface guidelines I think will suffice 👍 thank you!